src/components/Calculator.js

```diff
@@ -1,68 +1,59 @@
+/* eslint-disable no-unused-vars */
 /* eslint-disable no-console */
 import '../App.css';
-import React from 'react';
+import React, { useState } from 'react';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-    this.handleEvent = this.handleEvent.bind(this);
-  }
+function Calculator() {
```

Comment:
- [ ] Please, remember to use ES6 best practices for this. 🐶

Reply:
Hey there, could you please explain to me exactly what do you mean?😥

Reply:
Do you mean to instead of use a normal function to use an arrow function?

Reply:
Yes, I was talking about it. 😄

Reply:
Thanks for the feedbacks!